modules/ossm-control-plane-cli.adoc

```diff
@@ -19,7 +19,7 @@ You can deploy a basic `ServiceMeshControlPlane` from the command line.
 +
 [source,terminal]
 ----
-$ oc login https://<HOSTNAME>:6443
+$ oc login --username=<NAMEOFUSER> https://<HOSTNAME>:6443
```

Comment:
Some places mention port `8443` and some `6443`. Which one is correct? Both?

Reply:
Looks like both are correct from what I can gather.